styles/styles.css

```
@@ -65,47 +65,94 @@ body {
 
 .navigation {
   display: flex;
-  margin: 0 auto;
+  justify-content: space-between;
+  padding: 0 70px;
 }
 
-.site-navigation {
-  width: 617px;
-  margin: 0;
-  padding: 0;
+.navigation-list {
+  display: flex;
 }
 
 .user-navigation-list {
-  max-width: 318px;
-  margin-left: auto;
+  display: flex;
+  align-items: center;
 }
 
 .navigation-link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
   font-style: normal;
   font-weight: 700;
   font-size: 20px;
   line-height: 24px;
   color: #000000;
   text-decoration: none;
-  text-transform: capitalize;
+}
+
+.site-navigation {
+  display: flex;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000000;
+  align-items: center;
+  text-decoration: none;
+  flex-wrap: wrap;
+  padding: 20px 16px;
+}
+
+.text-button {
+  text-decoration: none;
+}
+
+.navigation-item {
+  padding: 20px 16px;
+}
+
+.nivigation-logo {
+  position: absolute;
+  height: 70px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.user-navigation-item {
+  padding: 22px 12px;
+}
+
+.user-navigation-button {
+  font-family: "PT Sans";
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  padding: 8px 50px;
+  color: #ffffff;
+  background-color: #756157;
+  border-radius: 4px;
+  flex-grow: 0;
+  border: none;
 }
 
 .main-index {
-  display: block;
   text-align: center;
   color: #000000;
   background-color: #ffffff;
 }
 
 .welcome-wraper {
```

```suggestion
.welcome-wrapper {
```
давай поправим везде опечатку